lib/main.dart

```diff
@@ -254,36 +253,6 @@ class SplashState extends ConsumerState<Splash> with WidgetsBindingObserver {
   }
 
   Future<void> checkLoggedIn() async {
-    /*bool jailbroken = false;
-    bool developerMode = false;
-    if (!kIsWeb && (Platform.isIOS || Platform.isAndroid)) {
-      jailbroken = await FlutterJailbreakDetection.jailbroken;
-      developerMode = await FlutterJailbreakDetection.developerMode;
-
-      if (!preferences.getHasSeenRootWarning() &&
-          (jailbroken || developerMode)) {
-        AppDialogs.showConfirmDialog(
-            context,
-            CaseChange.toUpperCase(AppLocalizations.of(context)!.warning,
-                StateContainer.of(context).curLanguage.getLocaleString()),
-            AppLocalizations.of(context)!.rootWarning,
-            AppLocalizations.of(context)!.iUnderstandTheRisks.toUpperCase(),
-            () async {
-              preferences.setHasSeenRootWarning();
-              checkLoggedIn();
-            },
-            cancelText: AppLocalizations.of(context)!.exit.toUpperCase(),
-            cancelAction: () {
-              if (!kIsWeb && Platform.isIOS) {
-                exit(0);
```

Comment:
I didn't manage the yes/no possibility within the popup because Apple can refuse the publication of the application if we "exit" the app : https://developer.apple.com/forums/thread/672098